src/main/resources/db/migration/V1__Initial.sql

```diff
@@ -122,7 +122,8 @@ create TABLE messages
     title           varchar(200)               NOT NULL,
     body            TEXT                       NOT NULL,
     date_time       timestamp WITH TIME ZONE   NOT NULL,
-    high_priority   boolean                    NOT NULL
+    high_priority   boolean                    NOT NULL,
+    unread          boolean                    NOT NULL
```

Comment:
Use `read` instead... as `unread` means `not read` so in order to mark a field as `read` currently you need to mark it as `not not read` ... it is much more clear to handle positive logic